app/controllers/merge_requests_controller.rb

```diff
@@ -119,6 +129,16 @@ def validate_response
       @merge_request.errors.add(:absorbing_organisation_id, I18n.t("validations.merge_request.absorbing_organisation_blank"))
       render previous_template
     end
+
+    if page == "confirm_telephone_number"
+      if merge_request_params[:telephone_number_correct].blank? && merge_request_params[:new_telephone_number].blank?
+        @merge_request.errors.add(:telephone_number_correct, I18n.t("validations.merge_request.telephone_number_correct_blank"))
+        render previous_template
+      elsif merge_request_params[:telephone_number_correct] == "0" && merge_request_params[:new_telephone_number].blank?
+        @merge_request.errors.add(:new_telephone_number, I18n.t("validations.merge_request.new_telephone_number_blank"))
+        render previous_template
+      end
+    end
```

Comment:
Not sure if it's because of this or something else, but I always have `Enter a new phone number` preselected before I answer that question. Any ideas?

Reply:
Also, should a validation be a different one if there is no current number and we only ask them to enter a new one?
Cause `Select to confirm or enter a new telephone number` sounds off if the user can't confirm anything 🤔